lib/email/receiver.rb

```diff
@@ -985,6 +1007,21 @@ def create_post(options = {})
       end
 
       user = options.delete(:user)
+
+      if options[:bounce]
+        options[:raw] = <<~MD
```

Comment:
This string still needs to be moved to `config/server.en.yml`.
Then use `options[:raw] = I18n.t("<key>", email: user.email, raw: options[:raw])`